File: Lista1/AluguelDeVeiculosEx1e2/src/apresentacao/Principal.java
package apresentacao;

import dados.*;
import negocio.*;

public class Principal{

    public static void main(String[] args) {
        
        Negocio sistema = new Negocio();
        Cliente c1 = new Cliente();
        Endereco e1 = new Endereco();
        Veiculo v1 = new Veiculo();
        Data d1 = new Data();

        c1.setNome("Joao");
        c1.setCpf("25695810382");

        e1.setRua("Joao colin");
        e1.setNumero(1020);
        e1.setBairro("Gloria");
        e1.setCidade("Joinville");
        e1.setEstado("SC");
        e1.setCep("1230932");
        c1.setEndereco(e1);

        v1.setModelo("Sedan");
        v1.setMarca("Fiat Pulse");
        v1.setAno(2022);
        d1.setDia(20);
        d1.setMes(05);
        d1.setAno(2022);
        v1.setData(d1);
        c1.setVeiculoAlugado(v1);

        sistema.setContClientes();
        sistema.setClientes(c1);

        sistema.setContAlugados();
        sistema.setVeiculos(v1);

        System.out.println(sistema.toString());

    }
}

File: Lista1/ex5/apresentacao/Principal.py
from dados.Cliente import Cliente
from dados.Data import Data
from dados.Endereco import Endereco
from dados.Veiculo import Veiculo
from negocio.Negocio import Negocio

class Principal:

    sistema = Negocio()
    cliente1 = Cliente()
    endereco1 = Endereco()
    veiculo1 = Veiculo()
    data1 = Data()

    cliente1.setNome("Joao")
    cliente1.setCpf("1239871")

    endereco1.setRua("Joao Colin")
    endereco1.setNumero(123)
    endereco1.setBairro("Gloria")
    endereco1.setCidade("Joinville")
    endereco1.setEstado("SC")
    endereco1.setCep("2938423")
    cliente1.setEndereco(endereco1)

    veiculo1.setModelo("Sedan")
    veiculo1.setMarca("Fiat Pulse")
    veiculo1.setAno(2022)
    data1.setDia(20)
    data1.setMes(5)
    data1.setAno(2022)
    veiculo1.setData(data1)
    cliente1.setVeiculoAlugado(veiculo1)

    sistema.setContAlugados()
    sistema.setVeiculo(veiculo1)
    sistema.setContClientes()
    sistema.setClientes(cliente1)



